lib/ui/platform_dispatcher.dart

```diff
@@ -1333,7 +1333,7 @@ class ViewConfiguration {
   /// relative to.
   ///
   /// If null, then this configuration represents a top level view itself.
-  final FlutterView? window;
+  final FlutterView? view;
```

Comment:
I am all in favor of this rename. As-is, this may be hard to land because presumably it breaks the framework and hence the auto-roller. You may have to do this in two stages where view/window exist in parallel, then you switch over the framework, and then you remove the window property.

Reply:
Is it possible to mark the property as deprecated?

Reply:
Yeah, you should be able to use `@Deprecated(<message>)`